spec/post_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(author_id: 1, title: 'Prayer', comments_counter: 0, likes_counter: 0) }
+
+  before { subject.save }
```

Comment:
- [ ] You did a great job on the tests @muhammedb8826, but according to the requirements, we need to test all the custom methods in every model. Hence we need to test the `update_post_counter` and `recent_comments` methods inside the `Post` model.

Reply:
Thank you for the feedback I will do that